examples/mirrorinstance.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: miniooperator.min.io/v1beta1
+kind: MirrorInstance
+metadata:
+  name: minio
+## If specified, MinIOInstance pods will be dispatched by specified scheduler.
+## If not specified, the pod will be dispatched by default scheduler.
+# scheduler:
+#  name: my-custom-scheduler
+spec:
+  selector:
+    matchLabels:
+      app: mc # Should match spec.metadata.labels
+  ## Add metadata to the all pods created by the StatefulSet
+  metadata:
+    labels:
+      app: mc # Should match spec.selector.matchLabels
+  ## Registry location and Tag to download MC image
+  image: minio/mc:RELEASE.2020-04-25T00-43-23Z
+  ## This is used to set configuration
+  ## Format name: MC_HOST_<alias>
+  ## Format value: https://<access_key>:<secret_key>@<minio_server_url>
+  env:
+    - name: MC_HOST_play
+      value: "https://Q3AM3UQ867SPQQA43P2F:zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG@play.min.io"
```

Comment:
can this automatically pick from the current deployment - or does it need to be separately deployed?

Reply:
if this is separately deployed, then how about a document on how to set this up?

Reply:
While we can use the current minio deployment, it would limit the mirror operation to be only for minio deployments, so I preferred the open approach where any source and target can be passed. I am adding details on how to set this up.

Reply:
Added the doc here https://github.com/nitisht/minio-operator/blob/mirror/docs/operator-fields.md